section-5/exercises/denysKozlovskyiSection5.js

```diff
@@ -0,0 +1,170 @@
+// Exercise 1:
+// 1. Write a function which will check if a user has a valid ticket ID and does user have 18+ to enter a club.
+// If condition is true then print 'User is allowed to enter the club!' if false then print 'User entrance is not allowed!'
+// Hint: use operator *&&* to check both conditions.
+function checkingUserIdTicket(ticketID, userAge){
```

Comment:
From this naming I understood that function check only user ticket id, but the function check if user is able to enter to the club or not by 2 parameters. So think about better name please.